src/meta_schedule/postproc/verify_gpu_code.cc

```diff
@@ -78,9 +126,11 @@ class VerifyGPUCodeNode : public PostprocNode {
           pass_list.push_back(tir::transform::LowerInitBlock());
           pass_list.push_back(tir::transform::PlanAndUpdateBufferAllocationLocation());
           pass_list.push_back(tir::transform::ConvertBlocksToOpaque());
-          pass_list.push_back(tir::transform::UnifyThreadBinding());
           pass_list.push_back(tir::transform::CompactBufferAllocation());
+          pass_list.push_back(tir::transform::Simplify());
```

Comment:
looks like the order of the passes we've got here is not very consistent with the current driver_api: https://github.com/apache/tvm/blob/main/src/driver/driver_api.cc#L249